### ISSUE TITLE
Vehicle Data Assertion Failed

### ISSUE THREAD

**Issue by iCollin** [smartdevicelink/sdl_core#3276]
## Bug Report

SDL core crashed while attempting to connect an app.

```
smartDeviceLinkCore: /home/livio/Desktop/collin/builds/dev_test/sdl_core/src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/vehicle_info_app_extension.cc:117: static vehicle_info_plugin::VehicleInfoAppExtension& vehicle_info_plugin::VehicleInfoAppExtension::ExtractVIExtension(application_manager::Application&): Assertion `(ext_ptr)' failed.
```

##### Expected Behavior
SDL Core never crashes

##### Observed Behavior
SDL Core crashed

##### OS & Version Information
* OS/Version: Ubuntu 18.04
* SDL Core Version: develop
* Testing Against: I was trying to connect an older Android Test Suite. (origin/core_6_0_0)

**Comment by iCollin** [comment]
Duplicate of https://github.com/smartdevicelink/sdl_core/issues/3108 (fixed by https://github.com/smartdevicelink/sdl_core/pull/3115)

**Comment by iCollin** [comment]
While this does seem like a duplicate of #3108, Core crashed with fix #3115. 

Reopening as this is an active issue.

**Comment by iCollin** [comment]
Could not reproduce.
